### ISSUE TITLE
IMPROVE - Scale Images on `SettingsView`

### ISSUE THREAD

**Issue by mikaelacaron** [mikaelacaron/Basic-Car-Maintenance#103]
## Describe the feature you want to improve
* When the dynamic type size of the `SettingsView` is at the largest size, you can see how the images don't scale (only the Contributors row does)
* Scale the icons as the dynamic type scales

![IMG_5493](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/9592d8ad-5295-4b80-bfeb-d0eba676ae81)

**Comment by hassan54** [comment]
@mikaelacaron please assign me this bug

**Comment by hassan54** [comment]
![Simulator Screenshot - iPhone Xʀ - 2023-10-13 at 16 35 52](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/6430106/427b0cff-87f4-42ae-84ed-5ef6749a76d4)
@mikaelacaron are they looking good now?

**Comment by mikaelacaron** [comment]
Yes that's the goal! @hassan54

**Comment by mikaelacaron** [comment]
@hassan54 Please open a PR with this fix! otherwise I'm going to release this issue and let someone else take it, because there's only 1 week left for Hacktoberfest

**Comment by hassan54** [comment]
@mikaelacaron PR https://github.com/mikaelacaron/Basic-Car-Maintenance/pull/221/

**Comment by mikaelacaron** [comment]
I haven't tested your PR but I was expecting something using ScaledMetric, as opposed to the way you implemented it

https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-scaledmetric-property-wrapper

**Comment by hassan54** [comment]
> I haven't tested your PR but I was expecting something using ScaledMetric, as opposed to the way you implemented it
> 
> https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-scaledmetric-property-wrapper

i have also implemented it using @ScaledMetric please see the SettingsView.swift file

**Comment by mikaelacaron** [comment]
Ahh I see now, but see the tutorial I linked, can you implement it by making a variable for the image size and make it relative to a dynamic type size, and use that variable for the size as opposed to having to multiple a variable for what's already there

**Comment by darrengillman** [comment]
Hi Mikaela, happy have a last minute crack at this if you'd like to get it in for the deadline.  Not checked the code yet, but if reasonably structured adding an @ScaledMetric affect to the images should be do-able.  He naively said... :)

**Comment by mikaelacaron** [comment]
@darrengillman You can do it! use this [article](https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-scaledmetric-property-wrapper), and relate it to the `.largeTitle` typography size
